icons/combine.svg

```diff
@@ -0,0 +1,19 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <rect width="4" height="4" x="2" y="2" />
+  <rect width="4" height="4" x="2" y="10" />
+  <rect width="4" height="4" x="2" y="18" />
+  <line x1="10" x2="14" y1="12" y2="12" />
+  <polyline points="12,10 14,12 12,14" />
+  <rect width="4" height="8" x="18" y="8" />
+  <line x1="18" x2="22" y1="12" y2="12" />
```

Comment:
```suggestion
  <circle cx="4" cy="12" r="2"/>
  <circle cx="4" cy="20" r="2"/>
  <circle cx="4" cy="4" r="2"/>
  <rect x="18" y="8" width="4" height="8" ry="2"/>
  <path d="M10 12h4"/>
  <path d="m12 10 2 2-2 2"/>
```

Reply:
Or more rounded blocks… (see pushed change)

Reply:
Feels a bit cramped no matter the radius, don't know how to fix it but having a bigger more visible arrow is needed IMO.

Reply:
I have an idea…

Reply:
Either something like (top left), or maybe better, adapt https://lucide.dev/icons?search=replace to `combine`. What do you think?

<img width="108" alt="Untitled" src="https://user-images.githubusercontent.com/7797479/232299180-808b3d2f-854c-48d4-b247-d65c7994df33.png">

Reply:
I like it :+1:

Reply:
Yeah, same here, I think using the opposite of `replace-all` should work pretty well.

Reply:
Now I’m thinking this should actually be `group` and then also have a counterpart `ungroup`…

Reply:
Not quite convinced about that, but there are some pretty good ideas for a group/ungroup icon pair in https://github.com/WordPress/gutenberg/issues/15602

Reply:
> ideas for a group/ungroup icon pair

Maybe that should be left for a separate PR then…